db/migrate/20211202105020_create_users.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.string :name
       t.string :photo
       t.text :bio
-      t.integer :postsCounter
+      t.integer :posts_counter
 
       t.timestamps
     end
```

Comment:
- [ ] I highlighted this because I don't see your models kindly work on associations between models `users`, `post`, `comment`, `like`
- [ ] User model should have a method that returns 3 most recent posts